src/barnetilsyn/Forside.tsx

```diff
@@ -21,15 +21,20 @@ import { isIE } from 'react-device-detect';
 import { useLokalIntlContext } from '../context/LokalIntlContext';
 import { Alert, Panel, Heading } from '@navikt/ds-react';
 import VeilederSnakkeboble from '../assets/VeilederSnakkeboble';
-import styled from "styled-components";
-
+import styled from 'styled-components';
+import {
+  erMellomStartenAvMaiOgSluttenAvAugust,
+  nåværendeÅr,
+} from '../utils/dato';
 
 const StyledAlert = styled(Alert)`
   margin-bottom: 2rem;
 `;
 
 const Forside: React.FC<any> = () => {
   const intl = useLokalIntlContext();
+  const erMellomMaiOgAugust = erMellomStartenAvMaiOgSluttenAvAugust();
```

Comment:
```suggestion
  const erMellomMaiOgAugust = dagensDatoErMellomMåneder(4, 7);
```
Ref. kommentaren nedenfor :)